src/common/network/TypesDef.h

```diff
@@ -0,0 +1,18 @@
+/* Copyright (c) 2018 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#pragma once
+
+#include <utility>
+
+#include "common/base/ThriftTypes.h"
+
+namespace nebula {
+
+// Host address type and utility functions
+using HostAddr = std::pair<IPv4, Port>;
```

Comment:
Why not define it in Base.h?

Reply:
Because some headers will only use this type. If defined in base.h, there are too many needless codes to  include for this type usage.